pkg/workloads/console/runner/runner.go

```diff
@@ -684,17 +680,17 @@ func rbHasSubject(rb *rbacv1.RoleBinding, subjectName string) bool {
 }
 
 // GetAttachablePod returns an attachable pod for the given console
-func (c *Runner) GetAttachablePod(csl *workloadsv1alpha1.Console) (*corev1.Pod, error) {
+func (c *Runner) GetAttachablePod(csl *workloadsv1alpha1.Console) (*corev1.Pod, string, error) {
```

Comment:
I think it'd be nicer to handle this by returning a struct. It'd be more compatible if we added more return args, but it'd also help keep context when reading a line in isolation.

That said, I think we should merge for now and I can adjust in my update to kubebuilder v2.

Reply:
Yeah true that would probably be nicer. But happy to defer and to just get this in for now.